<launch>
    <arg name="camera" default="color"/>
    <arg name="output" default="screen"/>
    <arg name="queue_size" default="5"/>

    <node pkg="stitchtron9000" type="extractor_node" name="extractor"
        output="$(arg output)" ns="$(arg camera)">
        <!-- parameters -->
        <param name="queue_size" type="int" value="$(arg queue_size)"/>
        <rosparam command="load" file="$(find stitchtron9000)/param/extractor.yml"/>
        <rosparam command="load" file="$(find stitchtron9000)/param/feature2d.yml"/>

        <!-- remappings -->
        <remap from="~image" to="image_rect_color"/>
        <remap from="~camera_info" to="camera_info"/>
        <remap from="~key_frame" to="key_frame"/>
    </node>
</launch> 
